### PR TITLE
Markdown Ads

### PR DESCRIPTION
**Pull request by hernanex3**
#258 
screenshot from 2018-08-01 12-00-47

### CODE REVIEW

File: lib/elephant_in_the_room/sites/post.ex [hunk 1]
```diff
@@ -4,7 +4,9 @@ defmodule ElephantInTheRoom.Sites.Post do
   alias Ecto.Changeset
   alias ElephantInTheRoom.Sites.{Post, Site, Category, Tag, Author}
   alias ElephantInTheRoom.{Repo, Sites}
-  alias ElephantInTheRoomWeb. Uploaders.Image
+  alias ElephantInTheRoomWeb.{Uploaders.Image}
```
Comment (juanbono) [inline lib/elephant_in_the_room/sites/post.ex]: ElephantInTheRoomWeb.Uploaders.Image is aliased twice

File: lib/elephant_in_the_room_web/templates/ad/form.html.eex [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+</fieldset>
+<% end %>
+
+<%= if assigns[:ad] do %>
+  <%= link to: site_ad_path(@conn, :delete, @site.name, @ad.name),
+          method: :delete, data: [confirm: "Esta seguro que quiere borrar la promocion?"],
+          class: "uk-button uk-button-danger uk-width-1-1" do %>
+    <i class="fas fa-trash"></i>
+  <% end%>
+<% end %>
```
Comment (juanbono) [inline lib/elephant_in_the_room_web/templates/ad/form.html.eex]: add newline

**Comment by hernanex3** [comment]
@juanbono I have just checked out https://hexdocs.pm/phoenix/contexts.html and I would prefer to make the "Context" changes globally in other PR (#406), because this is related to how we inserted everything in one context.

**Comment by juanbono** [comment]
Ok, I agree. It's not a trivial change.

**Pull request merged**